web_scraper/web_scraper/spiders/savantlynet.py

```diff
@@ -1,18 +1,55 @@
+# import scrapy
+# from scrapy.linkextractors import LinkExtractor
+# from scrapy.spiders import CrawlSpider, Rule
+
+
+# class SavantlynetSpider(CrawlSpider):
+#     name = "savantlynet"
+#     allowed_domains = ["savantly.net"]
+#     start_urls = ["https://savantly.net"]
+
+#     rules = (Rule(LinkExtractor(allow=r"Items/"), callback="parse_item", follow=True),)
+
+#     def parse_item(self, response):
+#         item = {}
+#         #item["domain_id"] = response.xpath('//input[@id="sid"]/@value').get()
+#         #item["name"] = response.xpath('//div[@id="name"]').get()
+#         #item["description"] = response.xpath('//div[@id="description"]').get()
+#         return item
+
```

Comment:
Let's keep the original class name for consistency.  
Also, let's remove code if it's not needed, instead of commenting it out.

Reply:
i'm commenting cause i will work to upgrade it. then i will remove the unnecessary  codes